src/Libraries/Microsoft.Extensions.AI/ChatCompletion/FunctionInvokingChatClient.cs

```diff
@@ -890,7 +958,7 @@ internal FunctionInvocationResult(bool shouldTerminate, FunctionInvocationStatus
         public Exception? Exception { get; }
 
         /// <summary>Gets a value indicating whether the caller should terminate the processing loop.</summary>
-        internal bool ShouldTerminate { get; }
+        public bool ShouldTerminate { get; }
```

Comment:
Remove the word "Should"? We just call this `Terminate` on FunctionInvocationContext.